index.html

```diff
@@ -35,12 +35,12 @@ <h1 id="name">I'm Suleiman Welcome To My Portfolio Website!</h1>
              </p>
         
         
-            <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
+            <div><a class="connect" href="#">LET'S CONNECT</a></div>
```

Comment:
- Hi @shaaibu7 and @boniface193, I believe this is what @OyePriscilla meant.  When the color contrats of your website was tested using [ aditus tool](https://www.aditus.io/button-contrast-checker/), some of the text and buttons were failing. Can we ensure to fix this? 

Here is the result of the test on some of your buttons 👇🏽
| LET'S CONNECT button 👇🏽  |
| :---:   | 
|![firstf](https://user-images.githubusercontent.com/99246359/207810731-f60b8608-22e1-4de7-9cfe-45b13a468107.PNG)  |

|Result of your Project's Logo 👇🏽  |
| :---:   | 
|![faitw](https://user-images.githubusercontent.com/99246359/207811072-e14ab05b-352c-4b61-8102-59daea26f949.PNG)  |

⚠️ These failures simply mean the colors of the text are too dull. Can you use a darker shade of the current colour? That way, you will have fixed the color contrast issues on your text.

_If you look closely, you will notice that the color of your name(logo) on hover is passing, that's because the color contrast is good._ 👍🏽